Simplenote/NoteListViewController.swift

```diff
@@ -2,10 +2,13 @@ import Foundation
 import SimplenoteSearch
 
 
-// MARK: - NoteListSearchDelegate
+// MARK: - NotesControllerDelegate
 //
-protocol NoteListSearchDelegate: AnyObject {
-    func notesListViewControllerDidSearch(_ query: SearchQuery?)
+protocol NotesControllerDelegate: AnyObject {
```

Comment:
I've kind of aligned the three delegates (so that they don't really have the word `View`):

- **TagsControllerDelegate**
- **NotesControllerDelegate**
- **EditorControllerDelegate**
- **EditorControllerNoteActionsDelegate**
- **EditorControllerTagActionsDelegate**

If you don't mind let's keep them this way! I've got in mind renaming `NoteListViewController` > `NotesViewController` and `TagListViewController` > `TagsViewController` since **FOREVER**!

🤟🤟🤟